Make_Tables/showtables.py

```diff
@@ -0,0 +1,13 @@
+from mysqlconnect import mydb
+
+mycursor = mydb.cursor()
+mycursor.execute("use sac_data")
+mycursor.execute("show tables")
+tables = mycursor.fetchall()
+for i in tables:
+	print("Table Name:",i[0])
+	mycursor.execute("describe "+i[0])
+	desription = mycursor.fetchall()
+	for j in desription:
+		print(*j)
+	print()
```

Comment:
- [ ] add newline at end of file
See reason : (https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file)